src/test/resources/kg/storages/s3.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
maybe it could be:
```
 "@id": "{myId}",
  "@type": "S3Storage",
  "default": false,
  "bucket": "{bucket}",
  "endpoint": "{endpoint}",
  "accessKey": "{accessKey}",
  "secretKey": "{secretKey}"
}
```

and then in the StoragesSpec you can just use the replacements. It seems a bit easier to read.